existing worlds not loading with v2.0.0

```
simplevoidworld:void -> 15
[21:31:10] [Server thread/INFO]: Dimension Id Info Saved to File
[21:31:12] [Server thread/INFO]: [STDOUT]: [transferchestmod]:(INFO) => DESERIALIZING
[21:31:12] [Server thread/INFO]: [STDOUT]: [transferchestmod]:(INFO) => Inventory: 0 air, 0 air, 0 air, 0 air, 0 air, 0 air, 0 air, 0 air, 0 air, 0 air
[21:31:12] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
```

and when generating a new SinglePlayer Map it is crashing with the following Crash-Log:
```
---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 1/29/21 9:45 PM
Description: Exception in server tick loop

org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at net.minecraft.class_3898.method_17228(class_3898.java:716)
	at net.minecraft.class_3898.method_20583(class_3898.java:388)
	at net.minecraft.class_3898$$Lambda$13798/479756303.accept(Unknown Source)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
	at java.util.Iterator.forEachRemaining(Iterator.java:116)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:502)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at net.minecraft.class_3898.method_17242(class_3898.java:385)
	at net.minecraft.class_3215.method_17298(class_3215.java:321)
	at net.minecraft.class_3218.method_14176(class_3218.java:710)
	at net.minecraft.server.MinecraftServer.method_3723(MinecraftServer.java:567)
	at net.minecraft.class_1132.method_3748(class_1132.java:83)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at net.minecraft.server.MinecraftServer$$Lambda$12560/431872709.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Variable modifier method addModDataVersions(Lnet/minecraft/class_2487;)Lnet/minecraft/class_2487; in dfubuddy.mixins.json:ChunkSerializerMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap supernatural-crops-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:408)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
	... 28 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1696227352 bytes (1617 MB) / 6514278400 bytes (6212 MB) up to 9544663040 bytes (9102 MB)
	CPUs: 16
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx10240m -Xms256m -XX:PermSize=256m
	Fabric Mods: 
		adorn: Adorn 1.13.0+1.16.5
		advanced_runtime_resource_pack: Runtime Resource Pack 0.3.2
		advancements-enlarger: Advancements Enlarger 0.2.3
		advdebug: Advancements Debug 2.2.0
		adventurez: AdventureZ 1.1.5
		ag4tr: Ag4tr 0.1.5.9
		allure: Allure 1.2.8
		amecsapi: Amecs API 1.1.1+mc1.16-rc1
		angerable-patch: Angerable Patch 1.1.0-1.16.4
		anvil_fix: AnvilFix 1.2.1-1.16
		appleskin: AppleSkin 1.0.11
		appliedenergistics2: Applied Energistics 2 8.1.0-alpha.13
		aquarius: Aquarius 1.0.3
		architectury: Architectury 1.5.104
		arstheurgia: Ars Theurgia 1.0.6
		artifacts: Artifacts 2.3.0+fabric
		artifice: Artifice 0.13.2+1.16.3
		artofalchemy: Art of Alchemy: Memoriam 1.0.0-rc4+1.16.4
		astromine: Astromine 1.11.5+fabric-1.16.5
		astromine-core: Astromine: Core 1.11.5+fabric-1.16.5
		astromine-discoveries: Astromine: Discoveries 1.11.5+fabric-1.16.5
		astromine-foundations: Astromine: Foundations 1.11.5+fabric-1.16.5
		astromine-technologies: Astromine: Technologies 1.11.5+fabric-1.16.5
		astromine-transportations: Astromine: Transportations 1.11.5+fabric-1.16.5
		audiooutput: AudioOutput 0.0.3
		authme: Auth Me 1.4.0
		autoconfig1u: Auto Config v1 Updated 3.3.1
		automated_crafting: Automated Crafting 1.3.20+MC1.16.2-1.16.5
		automotons: Automotons 1.0.1
		bambootweaks: Bamboo Tweaks unspecified
		battletowers: Battle Towers 3.3.2-1.16.4
		bebooks: Better Enchanted Books 1.2.1
		bedspreads: Bedspreads 1.3.1-1.16.4
		beebetter: Bee Better 1.8.0-1.16.4
		bettercaves: YUNG's Better Caves 1.16.2-1.0.3
		betterend: Better End 0.8.8-beta
		bettermineshafts: YUNG's Better Mineshafts 1.16.4-1.0
		betterslabs: Betterslabs 1.1.4
		bitsandchisels: BitsAndChisels 2.0.0
		blade: Blade 2.0.0
		blast: BLAST 1.5
		blockshifter: Blockshifter 1.3.2+fabric-1.16.4
		blockus: Blockus 2.0.2+1.16.4
		blue_endless_jankson: jankson 1.2.0
		boringbackgrounds: Boring Backgrounds 3.0.2+1.16.3
		byg: BYG 1.1.3
		campanion: Campanion 1.3.2
		can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.0.2
		cardinal-components: Cardinal Components API 2.7.10
		cardinal-components-base: Cardinal Components API (base) 2.7.10
		cardinal-components-block: Cardinal Components API (blocks) 2.7.10
		cardinal-components-chunk: Cardinal Components API (chunks) 2.7.10
		cardinal-components-entity: Cardinal Components API (entities) 2.7.10
		cardinal-components-item: Cardinal Components API (items) 2.7.10
		cardinal-components-level: Cardinal Components API (world saves) 2.7.10
		cardinal-components-scoreboard: Cardinal Components API (scoreboard) 2.7.10
		cardinal-components-util: Cardinal Components API (utilities) 2.7.10
		cardinal-components-world: Cardinal Components API (worlds) 2.7.10
		carrier: Carrier 1.6.1
		cavebiomes: Cave Biomes 0.5.1
		ce_foodstuffs: Common Expansion: Foodstuffs 1.1.2
		chainmail: Chainmail 0.5.0
		chains: Chains-Link 1.16.2-2.0.1
		charm: Charm 2.2.2
		charmonium: Charmonium 2.2.2
		cherishedworlds: Cherished Worlds 1.3.0-1.16.4
		chunkpregen: Fabric Chunk Pregenerator 0.3.3
		cinderscapes: Cinderscapes 1.3.2
		cleancut: CleanCut mc1.16-2.1-fabric
		cloth-api: Cloth API 1.4.9
		cloth-armor-api-v1: Cloth Armor API v1 1.4.9
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.4.9
		cloth-common-events-v1: Cloth Common Events v1 1.4.9
		cloth-config2: Cloth Config v4 4.8.3
		cloth-datagen-api-v1: Cloth Datagen v1 1.4.9
		cloth-durability-bar-api-v1: Cloth Durability Bar API v1 1.4.9
		cloth-dynamic-registry-api-v1: Cloth Dynamic Registry API v1 1.4.9
		cloth-scissors-api-v1: Cloth Scissors API v1 1.4.9
		cloth-utils-v1: Cloth Utils v1 1.4.9
		com_electronwill_night-config_core: core 3.6.2
		com_electronwill_night-config_toml: toml 3.6.2
		com_github_nathanpb_safer: Safer 1.1.0
		com_github_therandomlabs_trlutils-config: TRLUtils-Config master-SNAPSHOT
		com_github_therandomlabs_trlutils-fabric: TRLUtils-Fabric 1.16-SNAPSHOT
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		com_udojava_evalex: EvalEx 2.5
		completeconfig: CompleteConfig 0.11.0
		compressedblocks: Compressed Blocks 1.0
		computercraft: CC:T for Fabric 1.91.2
		conjuring: Conjuring 0.1.3-beta
		controlling: Controlling For Fabric 1.1.2
		cotton: Cotton 1.1.0
		cotton-cauldron: Cotton Cauldron 1.1.0
		cotton-commons: Cotton Commons 1.0.4
		cotton-config: Cotton Config 1.0.0-rc.7
		cotton-datapack: Cotton Datapack 1.0.7
		cotton-logging: Cotton Logging 1.0.0-rc.4
		cotton-player-events: Cotton Player Events 1.0.2
		couplings: Couplings 1.3.3
		crookedcrooks: Crooked Crooks 1.3.3+1.16.4
		croptopia: Croptopia 1.2.0
		crowdin-translate: CrowdinTranslate 1.2
		crumbs: Crumbs 1.0.6
		crusade: Crusade 1.2.0
		cullleaves: Cull Leaves 2.0.0
		damagetilt: Damage Tilt 1.16-fabric-0.1.1
		dankstorage: Dank Storage 1.9a-1.16.3
		darkpaintings: Dark Paintings 3.0.0
		dawn: Dawn API 1.6.1
		daylightdimension: Daylight Dimension 0.0.2
		decimal_coins: Decimal Coins 1.16.2-1.2.0
		decorative: Decorative 3.0.0
		dehydration: Dehydration 1.0.8
		diggusmaximus: Diggus Maximus 1.4.2-1.16.3
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.2
		dispenser_configurator: Dispenser Configurator 1.0.1+MC1.16.4-1.16.5
		dml-refabricated: Deep Mob Learning: Refabricated 0.3.1-beta5
		doorsofinfinity: Doors of Infinity 1.0.9-1.16.4
		dwarfcoal: Dwarf Coal 1.16.2-1.0.0
		dynamicfps: Dynamic FPS 2.0.1
		exotic-art-core: Exotic Art Core Content Pack 1.0.379
		exotic-art-tech: Exotic Art Tech Pack 1.2.379
		exotic-matter: Exotic Matter 2.12.381
		expandedstorage: ExpandedStorage 5.6.32
		extraalchemy: Extra Alchemy 1.3.1
		extradoors: ExtraDoors 1.8.2-1.16.2
		ezpas: Ezpas 1.2.3-1.16.2
		fabric: Fabric API 0.30.0+1.16
		fabric-api-base: Fabric API Base 0.2.0+ab87788d3a
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.0.10+ca58154a7d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.10+ca58154a7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-diagonal-panes: Fabric Diagonal Panes 0.4.0
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.2+ca58154a7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-furnaces: Fabric Furnaces 1.3.3-1.16.4
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.3+ca58154a7d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.2+ca58154a7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-language-kotlin: Fabric Language Kotlin 1.4.21+build.1
		fabric-language-scala: Fabric Language Scala 1.0.0
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.3+ca58154a7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.1+ca58154a7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabricloader: Fabric Loader 0.11.1
		facility: Facility 0.12.204
		fallflyinglib: FallFlyingLib 1.1.0
		falling_tree: FallingTree 2.10.0
		fatxporbs: Fat Experience Orbs 0.0.9
		fermion: Fermion 2.9.218
		fermion-gui: Fermion GUI 2.5.218
		fermion-modkeys: Fermion Modifier Keys 1.10.218
		fermion-orientation: Fermion Orientation 1.2.218
		fermion-varia: Fermion Varia 2.6.218
		fiber: fiber 0.23.0-2
		fiber2cloth: Fiber To Cloth 3.0.1
		flamingo_ooo: Flamingo, oh, oh, oh... 1.0.4-MC1.16.3-fabric
		flonters: Flonters 1.1.1+1.16.2
		flora-doubling: Flora Doubling 1.3.0+1.16.4-fabric
		fluidity: Fluidity 0.16.204
		fluidphysics: Fluid Physics 1.3.9+fabric-1.16.4
		fonthack: Font Hack 2.4.51
		framed: Framed 1.0.2
		frex: FREX 4.5.201
		gameinfo: GameInfo 2.0.1 1.16.4
		gateofbabylon: Gate Of Babylon 1.2.1-1.16.2
		geckolib3: Geckolib 3.0.13
		givemehats: Give Me Hats! 1.4
		glassdoor: Glass Doors 1.5.0-1.16
		glassential: Glassential 1.16.4-1.1.5
		goml: Get Off My Lawn 1.2.3-beta
		grindenchantments: Grind Enchantments 1.1.3+1.16.3
		hard-hat-steve: Hard Hat Steve 1.2.0
		harvest: Harvest 1.2.12
		harvest_scythes: Harvest Scythes 1.0.9
		hctm-base: HCTM Base 3.5.1
		healthoverlay: Health Overlay 3.6.14
		horseinfo: Horse Info 0.2.2-1.16.1
		hplus: Hopper Plus 1.0.10
		iamverysmart: I Am Very Smart 2.1.0+mc1.16.1
		identity: Identity 1.11.5-beta-1.16.4
		ignis-fatuus: Ignis Fatuus 1.1.3
		imm_ptl_core: Immersive Portals Core 0.68
		immersive_portals: Immersive Portals 0.68
		improved-stations: Improved Stations 2.0.3
		indrev: Industrial Revolution 1.7.15-BETA
		inmis: Inmis 2.1.1-1.16.4
		inventorysorter: Inventory Sorter 1.7.6-1.16
		io_github_onyxstudios_cardinal-components-api: Cardinal-Components-API 2.7.9
		iron-jetpacks: Iron Jetpacks 0.2.5
		jankson: Jankson 3.0.1+j1.2.0
		jargm: Just Another Redstone Gate Mod 1.2.0
		java: Java HotSpot(TM) 64-Bit Server VM 8
		javax_vecmath_vecmath: vecmath 1.5.2
		json-model-extensions: JSON Model Extensions 1.19.172
		jumploader: Jumploader 2.1.2
		kibe: Kibe 1.6.3-BETA
		ktdatataglib: KtDataTagLib 1.4.0
		kubejs: KubeJS 1.5.5
		kyrptconfig: Kytpt Config 1.1.6-1.16
		lakeside: Lakeside 1.0.3+1.16.4
		lambdynlights: LambDynamicLights 1.3.4+1.16
		lamps: The Lamps! 1.1
		lapisreserve: Lapis Reserve 1.0.6
		launchersmod: Launchers 3.1.0
		libaddict: libaddict 0.2.1
		libblockattributes: LibBlockAttributes 0.8.4
		libblockattributes_core: LibBlockAttributes (Core) 0.8.5
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.8.5
		libblockattributes_items: LibBlockAttributes (Items) 0.8.5
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 3.3.2+1.16.4
		libmultipart: LibMultiPart 0.4.3
		libmultipart_deps_container: LibMultiPart (Dependency Container) 0.4.3
		libnetworkstack: Lib Network Stack 0.4.3
		library_of_exile: Library of Exile 1.0.6
		libstructure: LibStructure 1.5
		lightoverlay: Light Overlay 5.6.1
		lithium: Lithium 0.6.1
		magna: Magna 1.4.3-1.16.2
		mcdict: MCDict 2.0.1+1.16.4
		mechanix: Mechanix 3.3.0
		megane: megane 3.1.2+1.16.4
		megane-api: megane-api 3.1.2+1.16.4
		megane-impl: megane-impl 3.1.2+1.16.4
		megane-runtime: megane-runtime 3.1.2+1.16.4
		mesh: Mesh 0.6.1-alpha.21+1.16.4
		mesh_api: Mesh API 0.6.1-alpha.21+1.16.4
		microconfig: Microconfig 1.0.5
		midnight-hats: Midnight Hats 2.0.1
		minecraft: Minecraft 1.16.5
		mo_glass: Mo Glass 1.5-MC1.16.5
		modern_industrialization: Modern Industrialization 0.3.11
		modmenu: Mod Menu 1.14.15
		more_tooltips: MoreTooltips 1.0.1
		moreberries: More Berries 1.3.4
		moretotems: More Totems 2.7.3
		mostructures: Mo' Structures 1.0.1+mc.1.16.4
		mousewheelie: Mouse Wheelie 1.6.3+mc1.16.4
		multi_item_lin: Multi Item Lin 1.0.0
		netherite_plus: Netherite Plus Mod 1.3.10-1.16.5
		no-potion-offset: No Potion Offset ${version}
		nookbuild: NookBuild 1.2.1
		notes: Notes 1.0.4
		nourish: Nourish 1.2.1
		okzoomer: Ok Zoomer 4.0.1+1.16.2
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_jetbrains_annotations: annotations 20.0.0
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.4.21
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.4.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.4.2
		org_joml_joml: joml 1.10.0
		oro_config: OroArmor Config unspecified
		packages: Packages 1.1.1
		patchouli: Patchouli 1.16.4-48-FABRIC
		pehkui: Pehkui 1.7.1+21w03a
		phonos: Phonos 0.1.1-beta
		phosphor: Phosphor 0.7.0+build.10
		piglib: Piglib 1.0.1
		plantinajar: Plant In A Jar 1.8.1
		plated: Plated 2.0.1
		platos: Plato's Transporters 1.6.4
		playerabilitylib: Pal 1.2.1
		pling: Pling 1.3.0
		polymorph: Polymorph 0.0.7-1.16.4
		programmerartinjector: Programmer Art Injector 1.2.1
		qcommon-cfg-core: qcommon-cfg-core 3.2.0
		qcommon-croco: Croco 2.1.1
		quickshulker: Quick Shulker 1.1.12-1.16
		reach-entity-attributes: Reach Entity Attribute 1.0.1
		reborncore: Reborn Core 4.6.4+build.117
		redbits: Red Bits 1.7.0
		repurposed_structures: Repurposed Structures 1.16.4-1.7.4
		respawnablepets: Respawnable Pets 1.16.5-1
		riverredux: River Redux 0.1.5
		rocks: This Rocks! 1.2.2
		roughlyenoughitems: Roughly Enough Items 5.9.1
		roughlyenoughitems-api: REI (API) 5.9.1
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.9.1
		roughlyenoughitems-runtime: REI (Runtime) 5.9.1
		roughlyenoughresources: Roughly Enough Resources 2.1.3
		rswires: RSWires 4.10.1
		rtree-3i-lite-fabric: rtree-3i-lite for Fabric 0.3.0
		sandwichable: Sandwichable 1.1.2
		screenshotclipboard: Screenshot to Clipboard 1.0.6
		secretrooms: Secret Rooms 0.9.2+1.16.2
		shulkerboxtooltip: Shulker Box Tooltip 2.3.4+1.16.4
		shulkerutils: shulkerutils 1.0.1-1.16
		silky_spawners: Silky Spawners 2.1.0+build.2-1.16
		simpledrawers: Simple Drawers 1.8.0-1.16.3
		simpleteleporters: Simple Teleporters 2.2.1
		simplevoidworld: Simple Void World 4.0.0
		slotlink: slotlink 2.2.0+1.16.4
		slotlock: SlotLock 1.0.1-BETA
		smithee: Smithee 0.2.3
		smootherbedrock: Smoother Bedrock 1.0.0
		soulcapture: Soul Capture 1.0.2
		spatialharvesters: Spatial Harvesters 0.10.6
		special-circumstances: Special Circumstances 1.8.218
		spruceui: SpruceUI 2.0.4+1.16
		stacc: Stacc 1.0.1
		staffofbuilding: Staff of Building 1.2.4-1.16.2
		statement: Statement 4.0.0+1.16.2
		static-content: Static Content 1.0.1-1.16.2
		staticdata: Static Data 1.1.2
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		stonecutter-tweaks: Stonecutter Tweaks 1.1.0-1.16
		strawdummy: Straw Dummy 1.4.0
		structure-helpers: Structure Helpers 3.0.0
		supernaturalcrops: Supernatural Crops 2.0.0
		tacocraft: TacoCraft 1.3.1+mc.1.16
		team_reborn_energy: Energy 0.1.1
		techreborn: Tech Reborn 3.7.3+build.192
		terraform-biome-builder-api-v1: Terraform Biome Builder API (v1) 1.0.0+build.4
		terraform-config-api-v1: Terraform Config API (v1) 1.0.0+build.4
		terraform-dirt-api-v1: Terraform Dirt API (v1) 1.1.0+build.4
		terraform-overworld-biome-extensions-api-v1: Terraform Overworld Biome Extensions API (v1) 1.0.0+build.4
		terraform-shapes-api-v1: Terraform Shapes API (v1) 1.0.1
		terraform-surfaces-api-v1: Terraform Surfaces API (v1) 1.0.0+build.4
		terraform-tree-api-v1: Terraform Tree API (v1) 1.0.1+build.4
		terraform-wood-api-v1: Terraform Wood API (v1) 1.0.1
		terrestria: Terrestria 2.1.6
		the_bumblezone: The Bumblezone - Fabric 1.16.5-2.2.17
		timeinabottle: Time in a Bottle 1.0.1-mc1.16.4-fabric
		tis3d: TIS-3D 1.6.0
		tis3d-additions: TIS-3D Additions 0.2.1+1.16.2
		tk_valoeghese_zoesteriaconfig: ZoesteriaConfig 1.3.2
		tooltipfix: ToolTip Fix 1.0.2-1.16
		totemmod: Wither Totems Mod 0.3.2
		transferchestmod: Transfer Chest 1.1.3
		trapexpansion: Trap Expansion 1.3.2
		traverse: Traverse 3.3.1
		trinkets: Trinkets 2.6.7
		truerooms: True Secret Rooms 1.1.2-hotfix
		tweed: Tweed API 3.0.0-beta.22
		vanilla-hammers: Vanilla Hammers 3.0.1-1.16.3
		vanilla_parts: Vanilla Parts 1.1.4+1.16.3
		vanilladeathchest: VanillaDeathChest 1.16.3-1.11.0.7-fabric
		vanillaexcavators: Vanilla Excavators 1.1.0
		villagerhats: Villager Hats fabric-1.16.4-1.0.1
		voyager: Voyager 1.0.0
		waila: Hwyla 1.9.22
		waystones: Waystones 1.1.1.1
		wild_explorer: Wild Explorer 1.3.1
		winged: Winged Mod 1.16.2-1.4.1
		wither_proof_block: Wither Proof Block 1.0.2
		wolveswitharmor: Wolves With Armor 1.5.2-1.16.2
		woodenhoppers: Wooden Hoppers 1.0.0
		woods_and_mires: Woods and Mires 1.0.4+1.16.3
		world_blender: World Blender 1.16.4-2.0.1
		wrenchable: Wrenchable 1.0.0+1.16.1
		xaerominimap: Xaero's Minimap 21.1.1
		xaeroworldmap: Xaero's World Map 1.11.10
		xb: Exotic Blocks 1.6.172
		xlpackets: XLPackets 1.1
		yalbm: Yet Another Lucky BLock Mod 1.1.7
		yosbr: YOSBR 0.1.1
		yungsapi: YUNG's API 1.16.4-Fabric-1
	Patchouli open book context: n/a
	Player Count: 1 / 8; [class_3222['MSandro'/122, l='ServerWorld minecraft:overworld x', x=104.50, y=4.00, z=-141.50]]
	Data Packs: secretrooms:server_pack, phonos:data_pack, vanilla (incompatible), Fabric Mods, global/cotton_generated
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fabric'
```

--- a/src/main/java/adudecalledleo/dfubuddy/mixin/ChunkSerializerMixin.java
+++ b/src/main/java/adudecalledleo/dfubuddy/mixin/ChunkSerializerMixin.java
@@ -2,18 +2,35 @@ package adudecalledleo.dfubuddy.mixin;
 
 import adudecalledleo.dfubuddy.impl.ModDataFixesInternals;
 import net.minecraft.nbt.CompoundTag;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.util.math.ChunkPos;
 import net.minecraft.world.ChunkSerializer;
+import net.minecraft.world.chunk.Chunk;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
-import org.spongepowered.asm.mixin.injection.ModifyVariable;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
 
 @Mixin(ChunkSerializer.class)
 public abstract class ChunkSerializerMixin {
+    // original injection by i5
+    // this seems to crash outside of dev tho, so we use an alternate @Inject
+    /*
     @ModifyVariable(method = "serialize",
                     at = @At(value = "INVOKE", target = "net/minecraft/nbt/CompoundTag.putInt(Ljava/lang/String;I)V", ordinal = 0),
                     name = "compoundTag")
     private static CompoundTag addModDataVersions(CompoundTag input) {
         ModDataFixesInternals.addModDataVersions(input);
         return input;
+    }
+     */
+
+    @Inject(method = "serialize",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/nbt/CompoundTag;putInt(Ljava/lang/String;I)V", ordinal = 0),
+            locals = LocalCapture.CAPTURE_FAILEXCEPTION)
+    private static void addModDataVersions(ServerWorld world, Chunk chunk, CallbackInfoReturnable<CompoundTag> cir,
+            ChunkPos chunkPos, CompoundTag compoundTag) {
+        ModDataFixesInternals.addModDataVersions(compoundTag);
     }
 }

--- a/src/main/java/com/fusionflux/supernaturalcrops/resource/SupernaturalCropsResources.java
+++ b/src/main/java/com/fusionflux/supernaturalcrops/resource/SupernaturalCropsResources.java
@@ -7,8 +7,9 @@ import com.fusionflux.supernaturalcrops.config.SupernaturalCropsConfig;
 import com.fusionflux.supernaturalcrops.item.SupernaturalCropsItems;
 import net.devtech.arrp.api.RRPCallback;
 import net.devtech.arrp.api.RuntimeResourcePack;
-import net.minecraft.block.Blocks;
+import net.minecraft.block.Block;
 import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
 import net.minecraft.util.registry.Registry;
 
 import static com.fusionflux.supernaturalcrops.SupernaturalCrops.id;
@@ -17,10 +18,10 @@ public class SupernaturalCropsResources {
     private static final RuntimeResourcePack RESOURCE_PACK = RuntimeResourcePack.create(SupernaturalCrops.MOD_ID + ":runtime");
 
     public static void register() {
+        RRPCallback.EVENT.register(a -> a.add(RESOURCE_PACK));
         registerLootTables();
         registerRecipes();
         registerBushResources(SupernaturalCropsBlocks.OreBushes.values());
-        RRPCallback.EVENT.register(a -> a.add(RESOURCE_PACK));
     }
 
     public static void registerBushResources(OreBush... bushes) {
@@ -30,10 +31,15 @@ public class SupernaturalCropsResources {
         }
     }
 
+    public static Identifier getBlockLootTableId(Block block) {
+        return id("blocks/" + Registry.BLOCK.getId(block).getPath());
+    }
+
     private static void registerLootTables() {
-        RESOURCE_PACK.addLootTable(id("blocks/embedded_abyss"), JLootTableFactory.createSimpleBlock(id("seeds_of_the_abyss")));
-        RESOURCE_PACK.addLootTable(id("blocks/scraped_stone"),
-                JLootTableFactory.createSimpleBlock(Registry.BLOCK.getId(Blocks.STONE)));
+        RESOURCE_PACK.addLootTable(getBlockLootTableId(SupernaturalCropsBlocks.EMBEDDED_ABYSS),
+                JLootTableFactory.createSimpleBlock(Registry.ITEM.getId(SupernaturalCropsItems.SEED_OF_THE_ABYSS)));
+        RESOURCE_PACK.addLootTable(getBlockLootTableId(SupernaturalCropsBlocks.SCRAPED_STONE),
+                JLootTableFactory.createSimpleBlock(Registry.ITEM.getId(Items.STONE)));
     }
 
     private static void registerBushLootTable(OreBush bush) {

--- a/src/main/java/com/fusionflux/supernaturalcrops/resource/SupernaturalCropsResources.java
+++ b/src/main/java/com/fusionflux/supernaturalcrops/resource/SupernaturalCropsResources.java
@@ -42,20 +42,21 @@ public class SupernaturalCropsResources {
     }
 
     private static void registerRecipes() {
-        RESOURCE_PACK.addRecipe(id("diamond_from_shards"), JRecipeFactory.create2x2(
-                "SS",
-                "SS",
+        RESOURCE_PACK.addRecipe(id("diamond_from_shards"), JRecipeFactory.create3x3(
+                "SSS",
+                "SSS",
+                "SSS",
                 Items.DIAMOND,
                 JRecipeFactory.key("S", SupernaturalCropsItems.DIAMOND_SHARD)));
-        RESOURCE_PACK.addRecipe(id("emerald_from_shards"), JRecipeFactory.create2x2(
-                "SS",
-                "SS",
+        RESOURCE_PACK.addRecipe(id("emerald_from_shards"), JRecipeFactory.create3x3(
+                "SSS",
+                "SSS",
+                "SSS",
                 Items.EMERALD,
                 JRecipeFactory.key("S", SupernaturalCropsItems.EMERALD_SHARD)));
-        RESOURCE_PACK.addRecipe(id("netherite_scrap_from_flakes"), JRecipeFactory.create3x3(
-                "FFF",
-                "FFF",
-                "FFF",
+        RESOURCE_PACK.addRecipe(id("netherite_scrap_from_flakes"), JRecipeFactory.create2x2(
+                "FF",
+                "FF",
                 Items.NETHERITE_SCRAP,
                 JRecipeFactory.key("F", SupernaturalCropsItems.NETHERITE_FLAKE)));
     }

